Remove duplicated CustomAttributeMixin include

tadeboro opened the pull request:
Source for ExtManagementSystem contains duplicated include of CustomAttributeMixin module. This commit removes include call that has been added last.

Commit that introduced the duplication: 
d25120a201b8c4d13b12919a99cd72e929bd00d6

/cc @gberginc

gberginc commented:
@tadeboro will it work if you remove the occurrence in L33 instead? I presume that this duplication was made because the initial `include` was hidden there.

carbonin commented:
Agree with @gberginc this is a good catch, but I think it's worth putting the include in a more sane place while you're here.

tadeboro commented:
I read the sources again and I could not find anything that would break if include was placed further down. Test seem to agree with me.

I updated the branch (and while I was at it, rebased it onto master).

As for why I removed the "wrong" include: to my C and python trained eye, both of those locations seem equal (in)sane, so I opted to remove the line that was added last.
